registry/kusama.json

```
@@ -166,6 +166,13 @@
         "shortDescription": "AIWeb3 Chinese community",
         "longDescription": "The mission of the AIWeb3 Chinese community is to continue building the Dotsama community, supporting Chinese content creators to focus on content creation related to Dotsama, establishing connections with different parachain projects, and providing the latest information about Dotsama. Chinese content creators can focus on researching and producing high-quality content, including dubbing, translation, and original content within Dotsama. Additionally, we will allocate more resources to support early stage content creators, helping them establish their audience and brand.",
         "isOrganization": true
-     }    
-    
+     },    
+     {
+        "address": "EAMbhBVW7SpJfH332uABbDNtsbNnaZBBXeGJwdfxkxkPm6o",
+        "name": "Dotcast",
+        "image": "https://github.com/D0tcast/opengov-delegate-registry/blob/8bc295ec75bdf4eb85ab771759fed0011fa14716/images/Dotcast%20Expect%20Chaos.png",
```

Only images that have been posted in the current repository and comply with our guidelines are permitted, could you change that link to this one:
```suggestion
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/Dotcast%20Expect%20Chaos.png",
```